packages/vm/src/checksum.rs

```diff
@@ -23,6 +23,12 @@ impl Checksum {
     pub fn to_hex(self) -> String {
         self.to_string()
     }
+
+    #[inline]
+    #[allow(clippy::len_without_is_empty)]
+    pub fn len(&self) -> usize {
+        32
+    }
```

Comment:
Wouldn't it be nicer to use `core::mem::size_of<Checksum>()` instead of having this function?

Reply:
Then maybe [size_of_val](https://doc.rust-lang.org/std/mem/fn.size_of_val.html) would be nicer?
It's really more of a nit-pick though.

Reply:
Yeah that's cool. I'll use that.